ci/nightly/pipeline.yml

```diff
@@ -0,0 +1,25 @@
+# Copyright Materialize, Inc. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+- id: sqlsmith
+  label: SQLsmith fuzzing
+  command: --query-count 10000
+  timeout_in_minutes: 240
+  plugins:
+    - ./ci/plugins/mzcompose:
+        config: test/smith/mzcompose.yml
+        run: smith-fuzz
+
+- id: perf-kinesis
```

Comment:
The default perf-kinesis is set to run for much longer than this. Should we update the defaults? Or will this pass by just not failing in that amount of time?

Reply:
Ah -- just seeing the `perf-kinesis` timeout now in the nightly run. I'm happy with either changing the defaults or passing new run params

Reply:
Ah, yeah, how long would you expect it to normally take? I think ~1hr is probably good for the nightly run to start. Also, is there log output while it is running? I was worried it was hung, because it only printed that it was selecting from the view once, but perhaps that is expected? (Not at all a problem, just curious.)

Reply:
That's expected -- it logs once when it begins to query and doesn't log again. I could add more messages in there!

I just ran the load test for ~24 hours, although 1 seems sufficient to surface any problems. Maybe the solution is I add a 24 hour run to release verification?

Reply:
(Created a PR for that: https://github.com/MaterializeInc/materialize/pull/2962)